code/game/objects/items/clown_items.dm

```diff
@@ -50,6 +59,18 @@
 	icon_state = "soapsyndie"
 	cleanspeed = 10 //much faster than mop so it is useful for traitors who want to clean crime scenes
 
+/obj/item/soap/sponge/bullet
+	name = "bullet sponge"
+	desc = "A sponge that is able to clean up any crime. Has some space lube mixed in and gives an faint smell of fear and lemons."
+	icon_state = "sponge"
+	armor = list("bullet" = 95)
+	cleanspeed = 5
+	bulletclean = TRUE
+
+/obj/item/soap/sponge/bullet/Initialize()
+	. = ..()
+	AddComponent(/datum/component/slippery, 120)
+
```

Comment:
No, this shouldn't be how you make a soap subtype more slippery! Add a new variable `/obj/item/soap` used on the `AddComponent` call in place of its second arg of 80.